modules/local/sra_fastq_ftp.nf

```diff
@@ -31,17 +32,17 @@ process SRA_FASTQ_FTP {
     script:    
     if (meta.single_end) {
         """
-        curl -L ${fastq[0]} -o ${meta.id}.fastq.gz
+        timeout $task.time bash -c 'until curl $args -L ${fastq[0]} -o ${meta.id}.fastq.gz; do sleep 1; done'; echo -e \\\\a

```

Comment:
What is the `echo -e \\\\a` doing here?

Reply:
This is from my initial testing, it alerts the console when it succeeds but I have removed it now